geoplot/geoplot.py

```diff
@@ -632,17 +632,22 @@ def __init__(self, df, **kwargs):
         self.ax = kwargs.pop('ax')
         self.extent = kwargs.pop('extent')
         self.projection = kwargs.pop('projection')
-        # TODO: init_axis() -> init_axis(ax)
-        self.init_axis()
+        # TODO: init_axes() -> init_axes(ax)
+        self.init_axes()
         self.kwargs = kwargs
 
-    def init_axis(self):
+    def init_axis(self, *args, **kwargs):
+        warnings.warn('Use Plot.init_axes instead of Plot.init_axis',
+                      DeprecationWarning, stacklevel=2)
+        self.init_axes(*args, **kwargs)
```

Comment:
This is safe to remove as you've changed the only call site (that I can see), so I'd just go ahead and do that